packages/perseus/src/interactive2/interactive-util.ts

```diff
@@ -106,24 +118,16 @@ const InteractiveUtil = {
     /**
      * Convert all function-or-array arguments to arrays of functions
      */
-    normalizeOptions: function (
-        arrayOptionNames: ReadonlyArray<string>,
-        options: Record<any, any>,
-    ): Record<any, any> {
-        // TODO(jack): Having to clone here is annoying; this
-        // function should really just modify this.state in place
-        // (and maybe be a function on MovableHelperMethods to get access
-        // to this.state), which would also be nicer because we could
-        // normalizeOptions once in this.modify
-        const result = _.clone(options);
-        _.each(arrayOptionNames, function (eventName) {
-            const funcOrArray = options[eventName];
+    normalizeOptions: function (options: Record<any, any>): Record<any, any> {
+        const result = {...options};
+        _.each(FUNCTION_ARRAY_OPTIONS, function (eventName) {
```

Comment:
I was looking through this and was concerned that we've made the set of options the same for all movables now whereas before it looks like they each had their own set of option names they supported.  

With this change am I reading it right that we basically support the union of all option names, but the set of `options` passed in will be restricted/correct for the give Movable subclass that calls this function?

Reply:
`undefined` values on `options` are ignored, so it's fine if a particular movable doesn't have handlers for certain events. The runtime behavior is the same as before.